migrations/7_VOTING_MACH-Genesis_Protocol.js

```diff
@@ -2,37 +2,42 @@
 /* eslint no-undef: "error" */
 const assert = require('assert')
 
-module.exports = async function (deployer) {
-  const DxGenesisProtocol = artifacts.require('DxGenesisProtocol')
+const getDaostackContract = require('../src/helpers/getDaostackContract')(web3, artifacts)
+
+
+module.exports = async function (deployer, network) {
+  const GenesisProtocol = artifacts.require('GenesisProtocol')
   const GenToken = artifacts.require('GenToken')
 
   // Deploy Genesis Protocol voting machine
-  // CAN ALSO REUSE
-  await deployGenesisProtocol(DxGenesisProtocol, GenToken, deployer)
+  if (network === 'development') {
```

Comment:
I think is better if we move this local development logic to the migration 2 with the other `development` specific contracts, and here we can just assume that `.deployed()` will work (in dev was deployed in 2, in other networks will use the address.

Does it make sense?